obspy/core/util/base.py

```diff
@@ -153,6 +153,7 @@ def create_empty_data_chunk(delta, dtype, fill_value=None):
         dtype = native_str(dtype)
     if fill_value is None:
         temp = np.ma.masked_all(delta, dtype=np.dtype(dtype))
+        temp.data[:] = np.nan
```

Comment:
This wont work for non-floating point dtypes I think. There is a doctest that tests this so CI should raise.